CHANGES.rst

```diff
@@ -28,6 +28,8 @@ Improvements
 - Add day of the week to room booking details modal and timeline (:issue:`5718`,
   :pr:`5743`)
 - Allow acceptance and rejection of editables in the editable list (:pr:`5721`)
+- Hide recurring booking tooltip from occurrences timeline (:issue:`5749`,
+  :pr:`5750`)
```

Comment:
I don't think a changelog entry is useful for this considering that it's the same version where the whole thing got introduced in the first place

Reply:
Removed.